starter-code/css/main.css

```diff
@@ -6,3 +8,177 @@ Green: #00B172
 White: #FFF
 
 */
+
+/* ---------------- General ---------------------------*/
+
+* {
+    font-family: Montserrat, Helvetica, Arial, sans-serif;
+    margin-top:0;
+    padding: 0; 
+    box-sizing: border-box;
+    color: 1A1A1A;
+}
+
+h4 {
+    font-size: 30px;
+    color: #00B172;
+}
+
+/* ---------------- Navbar ----------------------------*/
+
+.navbar {
+    z-index: 10;
+    background-color: rgba(255, 255, 255, 0.98);
+    position: fixed;
+    display: flex;
+    align-items: center;
+    height: 100px;
+    width: 100vw
```

Comment:
for the width: it's better to write 100% instead of 100vw (otherwise an horizontal scroll bar appear)